services/accounts/accounts.go

```diff
@@ -197,3 +290,73 @@ func (api *API) verifyPassword(password string) error {
 	_, err = api.manager.VerifyAccountPassword(api.config.KeyStoreDir, address.Hex(), password)
 	return err
 }
+
+func (api *API) GetAddressListForPath(password string, derivedFrom string, path string, pageSize int, pageNumber int) ([]*Address, error) {
+	derivedAddresses := make([]*Address, 0)
+
+	info, err := api.manager.AccountsGenerator().LoadAccount(derivedFrom, password)
+	if err != nil {
+		return nil, err
+	}
+
+	if pageNumber <= 0 || pageSize <= 0 {
```

Comment:
there is no max value for page size?

Reply:
I think there can be infinite number of pages. as in HD wallets. the UI can decide how many pages it wants to show?